static/templates/settings/notification_settings.hbs

```diff
@@ -69,6 +69,7 @@
                   {{#unless enable_sound_select}}
                   disabled
                   {{/unless}}>
+                    <option value="no sound">no sound</option>
                     {{#each page_params.available_notification_sounds}}
                     <option value="{{ this }}">{{ this }}</option>
```

Comment:
Also, can you try adding a commit to this PR to capitalization the names of the sounds in the labels?  I've been meaning to do that.  (E.g. call `toUpperCase` for the label part).